features/google/src/model/entities/user-activity.js

```diff
@@ -0,0 +1,13 @@
+import TimelineEvent from "./timeline-event";
```

Comment:
This seems something common enough to have been done already for Facebook; as a matter of fact, there's the "Activities" group here https://github.com/polypoly-eu/polyPod/blob/d91be81d075ca24b6507b06eb713fae84c56ef00/features/facebookImport/src/model/analyses/activities-analysis.js
They're essentially the same, except that "groups" are also defined. Have you considered creating this as a generic class so that it can benefit from common visualization tools?

Reply:
In an ideal case the "Activities"  analyses could simply work with entities of type "TimelineEvent" (or any object where timestamp returns a date object or a unix timestamp - something uniform) so it can be reused by the Google and Facebook features . That requires a bigger refactoring for the Facebook importer where the actual importers converts the date to the right format, and the analysis just gets the list of events from the account and groups them.